### DISCUSSION TITLE
Received the message "WARNING - We expected to get frame set in local storage but it was not" when trying to reference a SQL script in the MySQLOperator

### DISCUSSION THREAD

**Discussion by bmaury** [apache/airflow#23742] [General] [answered]
### Apache Airflow version

2.2.5

### What happened

When trying to use a file path to a sql file, rather than a SQL statement, in the MySQLOperator, the script errors our with the 'jinja2.exceptions.TemplateNotFound' exception, and I notice this warning as well.  See attached log. 
[mysql_operator_tester_test_mysql_2022-05-11T20_51_32.228555+00_00_6.log](https://github.com/apache/airflow/files/8704251/mysql_operator_tester_test_mysql_2022-05-11T20_51_32.228555%2B00_00_6.log)

### Operating System

Ubuntu 20.04.4 LTS

### Versions of Apache Airflow Providers

apache-airflow-providers-amazon==3.3.0
apache-airflow-providers-ftp==2.1.2
apache-airflow-providers-http==2.1.2
apache-airflow-providers-imap==2.2.3
apache-airflow-providers-mysql==2.2.3
apache-airflow-providers-sqlite==2.1.3

### Deployment

Other

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment, marked as answer]
The warning is irrelevant (and it's been fixed in 2.3.0). 
The root cause is the problem with missing template - you need to fix it and make sure the template is available. note that template files should be relative to the location of dags and you should place them in dag file rather than trying to set an absolute value).

See this FAQ for details: https://airflow.apache.org/docs/apache-airflow/stable/faq.html?highlight=templates#what-does-templatenotfound-mean

**Comment by potiuk** [comment]
Converted to discussion in case discussion is needed.

**Comment by bmaury** [comment]
Thanks for the quick response and pointer to the documentation.  I added in the template_searchpath variable , and this works as expected.  Note that once the template is located, the warning goes away as well.
